.github/workflows/test.yaml

```diff
@@ -140,3 +140,21 @@ jobs:
 
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
+
+  editorconfig:
+    name: Run editorconfig checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.5.2
+        with:
+          fetch-depth: 0
```

Comment:
`fetch-depth: 0` is for fetching entire git history/branches. It's not needed here, right?

Reply:
yeah, will update